msl_expressions/autogenerated/include/DomainBehaviour.h

```diff
@@ -41,6 +41,7 @@ class DomainBehaviour : public BasicBehaviour
 		void send(msl_helper_msgs::PassMsg& pm, int senderID);
 		void send(msl_helper_msgs::WatchBallMsg& wb);
 		void send(msl_helper_msgs::DebugMsg& dbm);
+		void sendAndUpdatePT(msl_actuator_msgs::MotionControl& mc);
```

Comment:
A comment on why to use it over `send(msl_actuator_msgs::MotionControl& mc)` the send would be nice.

Reply:
@StefanJakob @MarieRobots One of you add comments since you added this :)